.eslintrc

```diff
@@ -4,9 +4,10 @@
     "browser": true
   },
   "parser": "@typescript-eslint/parser",
-  "plugins": ["import", "react-hooks", "testing-library"],
+  "plugins": ["import", "react-hooks", "testing-library", "@chanzuckerberg/stories"],
   "extends": [
     "eslint:recommended",
+    "plugin:@chanzuckerberg/stories/strict",
```

Comment:
Strict includes
- no-csf-v2 (no csf v2 [functions]. Only vsf v3 [objects])
- no-ext-resources-in-stories (no images loaded over the interwebs, which cause visual testing flakiness)